### ISSUE TITLE
Can't control Border width

### ISSUE THREAD

**Issue by josedevb** [jxnblk/axs#26]
In the Border  component i can't modify the border width, like '3px solid #ffff', its only 1px default.

**Comment by jxnblk** [comment]
Yeah, it looks like the `css` prop is being assigned in the wrong order
